dashboard/test/models/ability_test.rb

```diff
@@ -61,6 +61,32 @@ class AbilityTest < ActiveSupport::TestCase
     assert ability.can?(:read, @login_required_script_level)
   end
 
+  test "as teacher" do
+    ability = Ability.new(create(:teacher))
+
+    assert ability.can?(:read, Game)
+    assert ability.can?(:read, Level)
+    assert ability.can?(:read, Activity)
+
+    refute ability.can?(:destroy, Game)
+    refute ability.can?(:destroy, Level)
+    refute ability.can?(:destroy, Activity)
+
+    assert ability.can?(:read, Section)
```

Comment:
Copied from "as student", with the `can?(:read, Section)` changed from `refute` to `assert`.